apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -734,10 +736,37 @@ export function buildMachine(
               always: 'not_accepting_paper',
             },
             resetting_state_machine_after_jam: {
-              entry: ['resetContext', 'endCardlessVoterAuth'],
-              after: {
-                [NOTIFICATION_DURATION_MS]: 'not_accepting_paper',
+              invoke: [pollPaperStatus(), pollAuthStatus()],
+              initial: 'reset_interpretation',
+              states: {
+                reset_interpretation: {
+                  entry: ['clearInterpretation'],
+                  on: {
+                    AUTH_STATUS_CARDLESS_VOTER: 'accepting_paper',
```

Comment:
Having separate flows helps WRT separating out the poll-worker-facing stuff so that we can temporarily reset language and color/size theme - that being said, we could potentially have both the non-English voter-facing text and the English-only poll-worker-facing text on the same screen.

No strong opinions making poll worker auth required though - tried to match the existing behaviour, but hearing your point about making sure a poll worker actually intervenes to replace the ballot correctly makes me lean toward making it required

Reply:
Ah didn't realize that. I was thinking consolidation would be good if the only reason to separate the steps was `xstate` semantics. But if there's a front end reason for it let's keep them separate.

Reply:
If I'm seeing this correctly in the video: we prompt a poll worker to insert the card, but don't strictly require it? I think that's fine.

Reply:
Yeah, that's right - sounds good!